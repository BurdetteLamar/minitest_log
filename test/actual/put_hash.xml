<log>
  <summary verdicts='0' failures='0' errors='0'/>
  <each_pair name='put_hash' class='Hash'>
    <![CDATA[
a => 0
b => 1
]]>
  </each_pair>
</log>
