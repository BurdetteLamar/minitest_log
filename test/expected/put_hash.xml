<log>
  <summary verdicts='1' failures='0' errors='0'/>
  <data name='put_hash' class='Hash' size='2' method=':each_pair'>
    <![CDATA[
a => 0
b => 1
]]>
  </data>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
